### DISCUSSION TITLE
Minimum system requirements?

### DISCUSSION THREAD

**Discussion by mariuszmaximus** [any1/wayvnc#89] [General]
I have imx8 platform 
I build system with bitbake 
After build wayvnc (v0.3.1) and run  I have error: 

```
 ./wayvnc
wl_registry@2: error 0: invalid version for global zxdg_output_manager_v1 (4): have 2, wanted 3
ERROR: Virtual Pointer protocol not supported by compositor.
ERROR: Failed to initialise wayland
```
Where to look for the problem?

**Comment by any1** [comment]
It seems that your compositor does not support the required protocols. Which compositor are you running?

**Reply by mariuszmaximus** [reply]
weston 9.0.0   ( Build: 9.0.0-34-ga1823f20+ ) 
Command line: /usr/bin/weston --log=/run/user/0/weston.log

Always after run ./wayvnc 
add line to  /run/user/0/weston.log
[16:24:34.821] libwayland: error in client communication (pid 817)

**Reply by mariuszmaximus** [reply]
My Linux is from bitbake 
https://www.nxp.com/docs/en/user-guide/IMX_YOCTO_PROJECT_USERS_GUIDE.pdf

**Reply by any1** [reply]
weston is not a wlroots based compositor, wayvnc will not work with it.

**Reply by mariuszmaximus** [reply]
Thank you !  Now it's all clear  
This is my first contact with Wayland

**Reply by danshick** [reply]
I think Weston has an RDP backend though. Might be what you're looking for.

**Reply by bmx666** [reply]
Hi @mariuszmaximus did you found any solution for Wayland RDP for yocto?
